public/app/features/annotations/editor_ctrl.ts

```diff
@@ -69,6 +72,14 @@ export class AnnotationsEditorCtrl {
     });
   }
 
+  /**
+   * Called from the react editor
+   */
+  onAnnotationChange(annotation: any) {
+    console.log('SAVE THE annotation!', annotation, this);
```

Comment:
@torkelo -- its too soon for a real review, but holler if this rings any bells for you:

![image](https://user-images.githubusercontent.com/705951/92217575-b8963b80-ee4c-11ea-8a16-89dfd81fc17f.png)

any hints on what to check?

lots going on with angular > react > angular madness!